pkg/document/model.go

```diff
@@ -75,6 +82,50 @@ func sortValueRows(valueRows []valueRow) {
 	})
 }
 
+func sortSectionedValueRows(sectionedValueRows []section) {
+	sortOrder := viper.GetString("sort-values-order")
+
+	if sortOrder != FileSortOrder && sortOrder != AlphaNumSortOrder {
+		log.Warnf("Invalid sort order provided %s, defaulting to %s", sortOrder, AlphaNumSortOrder)
+		sortOrder = AlphaNumSortOrder
+	}
+
+	for _, section := range sectionedValueRows {
+		sort.Slice(section.SectionItems, func(i, j int) bool {
```

Comment:
This whole sort section looks identical to the one used in `sortValueRows`. Could you factor that out into a standalone function that takes a `[]valueRow` and a `sortOrder` and uses it?